docs/source/model_doc/tapas.rst

```diff
@@ -130,6 +131,24 @@ for your environment):
         >>> config = TapasConfig('google-base-finetuned-wikisql-supervised')
         >>> model = TapasForQuestionAnswering.from_pretrained('google/tapas-base', config=config)
 
+And here is the equivalent code for TensorFlow (To run the TF model install the `tensorflow_probability dependency
+<https://github.com/tensorflow/probability`>_ for your environment):
```

Comment:
```suggestion
In TensorFlow, this can be done as follows (make sure to have installed the `tensorflow_probability dependency
<https://github.com/tensorflow/probability`>_ for your environment):
```

Reply:
Done